src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetMetaDataV1.java

```diff
@@ -249,4 +249,8 @@ public String getTableName(int column) throws SQLException {
   public int getColumnDisplaySize(int column) throws SQLException {
     return resultSetMetaData.getColumnDisplaySize(column);
   }
+
+  public SFResultSetMetaData getSfResultSetMetaData() {
```

Comment:
I don't like it, publishing next of our internal interfaces. But if we have to, let's mark it as internal.

Reply:
Fixed